css/properties/margin-block.json

```diff
@@ -6,13 +6,31 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/margin-block",
           "support": {
             "chrome": {
-              "version_added": "69"
+              "version_added": "69",
+              "flags": [
+                {
+                  "type": "runtime_flag",
+                  "name": "--enable-experimental-web-platform-features"
+                }
+              ]
```

Comment:
I've opened #6009 to address this more broadly, but we can do our part here to document this flag more consistently. Pending a decision on that, I'd suggest using this form for all the flags listed here:

```
"flags": [
  {
    "type": "preference",
    "name": "Experimental Web Platform features",
    "value_to_set": "Enabled"
  }
]
```

Reply:
I changed it. I also shared my opinion (spoiler - I do not agree with this) at https://github.com/mdn/browser-compat-data/issues/6009.

Reply:
Thank you for the changes and your input on #6009. I understand your disagreement and I am sympathetic to your point. I don't expect this to be the final form of the data and by aligning with data we already have, you're making it easier to get consistency with whatever we do end up with. I appreciate your help on this!